src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/test_vm_commands.py

```diff
@@ -748,7 +746,7 @@ class VMBootDiagnostics(ResourceGroupVCRTestBase):
     def __init__(self, test_method):
         super(VMBootDiagnostics, self).__init__(__file__, test_method, resource_group='cli_test_vm_diagnostics')
         self.vm_name = 'myvm'
-        self.storage_name = 'ae94nvfl1k1'
+        self.storage_name = 'clitestbootdiag20170227'
```

Comment:
Yes, once this is live, we can generate a random storage account name easily enough. The current problem is that making a randomized value work on playback is a lot of tedious work. If we plan to move away from VCR in the future, it might not be worth it to fix this.

The other problem you run into is that someone records the test in another subscription and the RG isn't removed. In that case, you have no way of removing the conflict name and thus you have to choose a new one.